.env.example

```
@@ -43,6 +43,12 @@ ANTHROPIC_API_KEY=
 OPENAI_API_KEY=
 GEMINI_API_KEY=
 MINIMAX_API_KEY=
+# Z.AI GLM models — intl (api.z.ai) / CN (open.bigmodel.cn); *_CODING keys are
+# for the subscription coding plans served via the Anthropic-compatible endpoints.
+ZAI_API_KEY=
+ZAI_CN_API_KEY=
+ZAI_CODING_API_KEY=
+ZAI_CN_CODING_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ZAI_CN_CODING_API_KEY key should go before the ZAI_CODING_API_KEY key

(UnorderedKey)